src/req.cpp

```diff
@@ -67,6 +67,7 @@ void printErrors()
                 fprintf(stderr, "%s:%s: %s\n", file->first.c_str(), e->first.c_str(), e->second.c_str());
             }
         }
+        exit (EXIT_FAILURE);
```

Comment:
OK, that sound good to me I have pushed a new commit doing exactly that. And some others of your suggestions below.

Note: About the exit value I had choosen '1' 'EXIT_FAILURE' because 'EXIT_SUCCESS' and 'EXIT_FAILURE' are the two only values recognized and guaranteed to work by ISO C and ISO C++. But in fact I do not know about any real life case where '2' will not work. Then as you prefer '2' it is also good for me.